examples/blogs__parcel_preprocessor/cypress/plugins/index.js

```diff
@@ -0,0 +1,88 @@
+const ParcelBundler = require('parcel-bundler')
+const path = require('path')
+
+/* global Promise */
+/* eslint-disable no-console */
+
+const bundleOnce = (filePath, outputPath) => {
+  const outDir = path.dirname(outputPath)
+  const outFile = path.basename(outputPath)
+
+  // seems parcel-bundler 1.7.0 does not want extension
+  // in the output filename
+  const basename = path.basename(outFile, path.extname(outFile))
+  const options = {
+    watch: false,
+    hmr: false,
+    outFile: basename,
+    outDir,
+  }
+  const bundler = new ParcelBundler(filePath, options)
+  return bundler.bundle()
+}
+
+const bundlers = {}
+
+const onFile = (file) => {
+  const { filePath, shouldWatch, outputPath } = file
+  if (!shouldWatch) {
+    console.log('bundle file once without watching to %s', outputPath)
+    return bundleOnce(filePath, outputPath).then(() => outputPath)
```

Comment:
Shouldn't this have the same logic as below where you check if there's a result and fail if not?